dm/syncer/syncer.go

```diff
@@ -2967,6 +2988,70 @@ func (s *Syncer) trackDDL(usedSchema string, trackInfo *ddlInfo, ec *eventContex
 	return nil
 }
 
+func (s *Syncer) trackOriginDDL(ev *replication.QueryEvent, ec eventContext, originSQL string) (map[string]map[string]struct{}, error) {
+	if originSQL == "BEGIN" || originSQL == "" || utils.IsBuildInSkipDDL(originSQL) {
```

Comment:
don't need param:`originSQL`, we can use strings.TrimSpace(string(ev.Query)) instead